src/compiler/scala/tools/nsc/transform/Delambdafy.scala

```diff
@@ -98,13 +101,21 @@ abstract class Delambdafy extends Transform with TypingTransformers with ast.Tre
       val isSerializable = samUserDefined == NoSymbol || samUserDefined.owner.isNonBottomSubClass(definitions.JavaSerializableClass)
       val addScalaSerializableMarker = samUserDefined == NoSymbol
 
+      val samBridges = logResultIf[List[Symbol]](s"will add SAM bridges for $fun", _.nonEmpty) {
+        userSamCls.fold[List[Symbol]](Nil) {
+          _.info.decls.collect {
+            case bridge: MethodSymbol if bridge hasFlag BRIDGE => bridge
+          }.toList
+        }
+      }
```

Comment:
`userSamCls.findMembers(0L, BRIDGE)` might also work here (we know the only base class is `Object` without bridges).

Reply:
Seems to work, and you're right that our base types are only the SAM trait and `Object`. I'll change it.